test/AutoRest.TestServerLowLevel.Tests/AccessibilityTests.cs

```diff
@@ -14,7 +14,7 @@ public class AccessibilityTests
         [Test]
         public void AccessibilityHonoredOnOperations()
         {
-            var client = typeof(AccessibilityClient);
+            var client = typeof(AccessibilityRestClient);
```

Comment:
Since we're planning to enable incremental HLC functionality in the future, `*RestClient` suffix seems better to me.